public/js/utils/sources-tree.js

```diff
@@ -4,11 +4,6 @@ const { isPretty } = require("../utils/source");
 
 const IGNORED_URLS = ["debugger eval code", "XStringBundle"];
 
-function isHiddenSource(source) {
-  const url = source.get("url");
-  return !url || url.match(/SOURCE/) || IGNORED_URLS.includes(url);
-}
```

Comment:
i'm sure you know this, but `isHiddenSource` is used in many places, which is one of the benefits of having it like that. 

e.g. do we show a source tab